source/features/checks-modal.tsx

```diff
@@ -0,0 +1,38 @@
+import {$} from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import delegate, {DelegateEvent} from 'delegate-it';
+
+import features from '../feature-manager.js';
+
+function onButtonClick({delegateTarget: delegate, target}: DelegateEvent): void {
+	// Only close if clicking outside of modal
+	if (delegate === target) {
+		$('[class^="Dialog__DialogCloseButton-"], [class*=" Dialog__DialogCloseButton-"]', delegate)?.click();
```

Comment:
Instead of an additional selector, can we just `target.dispatchEvent(new KeyboardEvent(simulate ESC somehow))` instead?